cmd/opentelemetry-collector/app/defaults/default_config.go

```diff
@@ -90,6 +93,10 @@ func createReceivers(zipkinHostPort string, factories config.Factories) configmo
 			},
 		},
 	}
+	jaeger.RemoteSampling = &jaegerreceiver.RemoteSamplingConfig{
```

Comment:
Should the OTC, when used as a Jaeger collector replacement, be exposing the agent ports?

Reply:
Good point, I have disabled it for now.